http-endpoint-example/README.md

```diff
@@ -0,0 +1,41 @@
+# How to download Met Office atmospheric model data using an HTTP endpoint
+A demonstration code example of downloading objects in response to SNS messages via an HTTP endpoint.
+
+By using the following you will be able to subscribe to a Amazon Web Services (AWS) Simple Notification Service (SNS) and start downloading objects.
+The running python script will accept notfications from the the SNS and download the objects according to the input paramters.
+
+
+## Prerequisites:
+
+  * API key
+  * Python 3 environment
+  * Open port
```

Comment:
This doesn't make any sense on its own - an open port on what? We should combine it with the following requirement for a server with a public address.